diff -Naur courier-imap-4.11.0.orig/imap/authenticate_auth.c courier-imap-4.11.0/imap/authenticate_auth.c
--- courier-imap-4.11.0.orig/imap/authenticate_auth.c	2012-10-09 11:07:43.255759323 +0400
+++ courier-imap-4.11.0/imap/authenticate_auth.c	2012-10-09 11:07:48.618933347 +0400
@@ -134,7 +134,7 @@
 	p=getenv(authservice);
 
 	if (!p || !*p)
-		p="imap";
+		p="courier-imap";
 
 	rc=auth_generic(p, authtype, authdata, login_callback, (void *)tag);
 	free(authtype);
diff -Naur courier-imap-4.11.0.orig/imap/imaplogin.c courier-imap-4.11.0/imap/imaplogin.c
--- courier-imap-4.11.0.orig/imap/imaplogin.c	2012-10-09 11:07:43.257759388 +0400
+++ courier-imap-4.11.0/imap/imaplogin.c	2012-10-09 11:07:48.618933347 +0400
@@ -354,7 +354,7 @@
 		p=getenv(authservice);
 
 		if (!p || !*p)
-			p="imap";
+			p="courier-imap";
 
 		rc=auth_login(p, userid, passwd, login_callback, (void *)tag);
 		courier_safe_printf("INFO: LOGIN FAILED, user=%s, ip=[%s]",
diff -Naur courier-imap-4.11.0.orig/imap/pop3login.c courier-imap-4.11.0/imap/pop3login.c
--- courier-imap-4.11.0.orig/imap/pop3login.c	2012-10-09 11:07:43.255759323 +0400
+++ courier-imap-4.11.0/imap/pop3login.c	2012-10-09 11:07:48.618933347 +0400
@@ -368,7 +368,7 @@
 						strcat(strcpy(authservice, "AUTHSERVICE"),getenv("TCPLOCALPORT"));
 						q=getenv(authservice);
 						if (!q || !*q)
-							q="pop3";
+							q="courier-pop3";
 
 						rc=auth_generic(q,
 							     authtype,
@@ -416,7 +416,7 @@
 				strcat(strcpy(authservice, "AUTHSERVICE"),getenv("TCPLOCALPORT"));
 				q=getenv(authservice);
 				if (!q || !*q)
-					q="pop3";
+					q="courier-pop3";
 
 				rc=auth_login(q, user, p, login_callback, NULL);
 				courier_safe_printf("INFO: LOGIN "
